resources/language/resource.language.nl_nl/strings.po

```diff
@@ -352,39 +352,39 @@ msgstr "Bevat {} en meer..."
 
 msgctxt "#30088"
 msgid "Browse related content"
-msgstr "Doorblader gerelateerde videos"
+msgstr "Blader door gerelateerde video's"
 
 msgctxt "#30089"
 msgid "Browse subgenres"
-msgstr "Doorblader subgenres"
+msgstr "Blader door subgenres"
 
 msgctxt "#30090"
 msgid "Browse through related video lists and discover more content."
-msgstr "Doorblader gerelateerde videos en ontdek meer inhoud."
+msgstr "Blader door gerelateerde video's en ontdek meer inhoud."
 
 msgctxt "#30091"
 msgid "Browse and manage contents that were exported to the Kodi library."
-msgstr "Doorblader en beheer geëxporteerde videos in de Kodi bibliotheek."
+msgstr "Blader door en beheer geëxporteerde video's in de Kodi bibliotheek."
 
 msgctxt "#30092"
 msgid "Search for content and easily find what you want to watch."
-msgstr "Zoek naar videos en vind met gemak wat je zocht."
+msgstr "Zoek naar video's en vind met gemak wat u zocht."
 
 msgctxt "#30093"
 msgid "Browse content by genre and easily discover related content."
-msgstr "Doorblader videos op basis van genre en ontdek met gemak gerelateerde inhoud."
+msgstr "Blader door video's gebaseerd op genre en ontdek meer gerelateerde inhoud."
 
 msgctxt "#30094"
 msgid "Browse recommendations and pick up on something new you might like."
-msgstr "Doorblader aangeprezen videos en kies iets dat je bevalt."
+msgstr "Blader door aangeprezen video's en kies iets dat je leuk lijkt."
 
 msgctxt "#30095"
 msgid "All TV Shows"
-msgstr "Alle tvprogramma's"
+msgstr "Alle TV Programma's"
 
 msgctxt "#30096"
 msgid "All Movies"
-msgstr "Alle films"
+msgstr "Alle Films"
```

Comment:
Films should be lowercase

Reply:
Disagree. The words in msgid also start with a capital letter.

Reply:
It's common to capitalise nouns in English, but it's also wrong. In Dutch, this isn't common and it is definitely wrong.

See https://www.grammarly.com/blog/capitalization-rules/